tern/analyze/default/container/image.py

```diff
@@ -23,10 +23,29 @@
 logger = logging.getLogger(constants.logger_name)
 
 
+def download_container_image(image_tag_string):
+    '''Download the docker image and convert it into oci format'''
+    try:
+        # extract the docker image
+        docker_image = image_tag_string
+        if 'docker://' in image_tag_string:
+            docker_image = image_tag_string.split('docker://')[1]
+        image_attr = general.parse_image_string(docker_image)
+        oci_image = 'oci://{0}/{1}'.format(
+            rootfs.working_dir, image_attr.get('name'))
+        docker_image = 'docker://{0}'.format(docker_image)
+        cmd = 'skopeo copy {0} {1}'.format(docker_image, oci_image)
```

Comment:
`tern.utils.rootfs` has a `shell_command` function that will take a command like `["skopeo", "copy"]` and some extra arguments like `docker_image` and `oci_image` and invoke it using `subprocess`. In `rootfs.py` the security linting error is suppressed, so the linter will not complain. Why not try this method?